### DISCUSSION TITLE
Provide an RSS/Atom feed to your changelog

### DISCUSSION THREAD

**Discussion by Hyask** [keygen-sh/community#10] [Ideas]
Hello there!

Do you think it could be possible to keep track of your changelog with a feed aggregator?

**Comment by ezekg** [comment]
This is a good suggestion. I'll see if I can add a feed soon.

**Comment by ezekg** [comment]
An RSS feed is now available here: https://keygen.sh/changelog/feed.xml

**Reply by Hyask** [reply]
Thanks, that's great!
